backend/src/models/service.ts

```diff
@@ -163,6 +163,17 @@ export class Service extends BaseEntity {
     }[];
   };
 
+  /** Shodan results */
+  @Column({
+    type: 'jsonb',
+    default: {}
+  })
+  shodanResults: {
```

Comment:
Does Shodan only give us a single CPE per service?

Reply:
It's possible that it does give multiple CPEs. This is a little odd since they only give one product/version for each service, but potentially more than one CPE. Not sure if there's a better way to handle this, thoughts?

Reply:
Yeah, I think it's fine in that case then.

Reply:
From the ones I've seen, they're sometimes different products.